_posts/samandalib/2020-08-26-first-post-samandalib.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+author: samandalib
+title: "Hesam's first post!"
+---
```

Comment:
This is the YAML header that tells our website how to make this look like a post